src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -181,9 +181,11 @@ sub-<label>/
 ```
 
 This file is RECOMMENDED as it provides easily searchable information across
-MEG-BIDS datasets for e.g., general curation, response to queries or batch
```

Comment:
I think MEG-BIDS references should be removed everywhere in the spec. It fractures the specification. But it's a separate thing of course.

Reply:
If you mean by that "there is **one** BIDS, not EEG-BIDS, MEG.BIDS, MRI-BIDS, ... ", then I agree and we could open an issue for it :-)

Reply:
yes agreed :)